### ISSUE TITLE
Allow users provide values for input slots in the constructor

### ISSUE THREAD

**Issue by m-novikov** [ilastik/lazyflow#309]
We can introduce logic in operators constructors to allow providing values as keyword arguments, it will result in something like this.
```
op = Operator(parent=self, Input1=value1, Input2=value2)
op.Output.value
```
For this, in slot constructor, we should parse `kwargs` and match them with `self.inputs`
As [operator constructor](https://github.com/ilastik/lazyflow/blob/55858dd85af8c08be30fe4da018b8651cbeb7803/lazyflow/operator.py#L204) right now only accepts two arguments(graph and parent) it would be pretty straightforward to implement.

**Comment by Tomaz-Vieira** [comment]
See related ranting here: https://github.com/ilastik/lazyflow/pull/293

TL;DR:
I'm not really comfortable with having that logic in the parent `Operator` class; the order in which the slots would be collected might affect something (like having the optional slots last, triggering several `setupOutput`s). Also, if we actually have the operators requirements explicitly listed in their `__init__`s, then we could eventually remove the "static" slot declarations.
